### ISSUE TITLE
WRF Ensemble Setup Script

### ISSUE THREAD

**Issue by phawbeck** [a2e-mmc/mmctools#107]
Create a WRF setup class that will create the running directories for a defined ensemble (say GFS, ERA-I, ERA5 w/ YSU and MYJ PBL) and then submission scripts allowing for all to be submitted at once, or individually. Only things that would need to be changed between case studies would be domain setup and run times. This can build off of the current wrf-setup.sh script but may be beneficial to be re-written in python to allow for calling other python codes.

**Comment by ewquon** [comment]
@phawbeck, this is what I'm envisioning:
```python
from mmctools.wrf.ensemble import Ensemble
wrfruns = Ensemble(
    '/path/to/sim/dir',
    start='2013-11-08',
    end='2013-11-09 12:00',
    reanalyses=['GFS','ERA-I','ERA5','NARR'], # default
    pbl_schemes=['YSU','MYJ','MYNN'], # default
)
wrfruns.setup() # run geogrid once, use mmctools.wrf.preprocessing to get all reanalysis datasets
wrfruns.submit(config='eagle') # or 'cheyenne', 'generic_pbs', 'generic_slurm', ...
```

**Comment by phawbeck** [comment]
Yeah, if we got that working, I think that would be one hell of a start. Additional options for things like SST, microphysics, or whatever can easily be added in from there.

**Comment by ewquon** [comment]
Cool, updating this for my reference. 
```python
from mmctools.wrf.ensemble import Ensemble
wrfruns = Ensemble(
    '/path/to/sim/dir',
    start='2013-11-08',
    end='2013-11-09 12:00',
    ref_lat=33.61054,
    ref_lon=-102.05054,
    reanalyses=['GFS','ERAInterim','ERA5','NARR'], # default
    pbl_schemes=['YSU','MYJ','MYNN'], # default
)
wrfruns.setup() # run geogrid once, use mmctools.wrf.preprocessing to get all reanalysis datasets
#wrfruns.adjust_SST()
wrfruns.submit_all(config='eagle') # or 'cheyenne', 'generic_pbs', 'generic_slurm', ...
```

**Comment by phawbeck** [comment]
This was addressed in #121 and is closed.
